### ISSUE TITLE
Do I need to change the prompt in step 3?

### ISSUE THREAD

**Issue by yuesam** [lllyasviel/Paints-UNDO#26]
Do I change the prompt in step 3 to the prompt generated in step 1? Thanks

**Comment by another-ai** [comment]
Nope, I think it's an error, in the huggingface version this line is different so the prompt is the same(https://huggingface.co/spaces/MohamedRashad/PaintsUndo/blob/main/app.py):
```
def interrogator_process(x):
    image_description = wd14tagger.default_interrogator(x)
    return image_description, image_description
```

**Comment by lllyasviel** [comment]
hi - please do not change codes. 

that huggingface space is wrong and gives worse results.

**Comment by another-ai** [comment]
> hi - please do not change codes.
> 
> that huggingface space is wrong and gives worse results.

Yes, but I think that part is correct... why step 3 must have a different prompt thank step 1 prompt?

**Comment by lllyasviel** [comment]
Because the beginning frames are not correspondent to the tags - if your tags are "1girl, dog, cat", the beginning frames without that dog and cat will suffer. Later frames are less influenced, though.

**Comment by MohamedAliRashad** [comment]
@yuesam That's my mistake, i though the stage 1 prompt will enhance stage 3 but i was clearly wrong.
I fixed it in the space, Happy building ^^
